lib/wasix/src/os/console/mod.rs

```diff
@@ -205,7 +205,34 @@ impl Console {
             }
         };
 
-        let wasi_opts = webc::metadata::annotations::Wasi::new(prog);
+        let command = if let Some(entrypoint) = pkg.entrypoint_cmd.as_deref() {
+            pkg.get_command(entrypoint).unwrap()
```

Comment:
Can we return an error here?

It'd be annoying if we start panicking on Edge because an oversight in `wasmer publish` meant we publish packages with a bad `package.entrypoint` field.

Reply:
I assumed that it was impossible for the entrypoint to not exist.